packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -798,22 +796,42 @@ function FirstTermDiscountCallout( { product }: { product: ResponseCartProduct }
 		// Do not display discount reason if a coupon is applied.
 		isCouponApplied( product )
 	) {
-		return null;
```

Comment:
There may be a reason I don't understand but what's the benefit of changing this `null` to an empty fragment?